docs/release/releases/release-1.7.md

```diff
@@ -6,22 +6,22 @@ The following table shows the preliminary dates for the `v1.7` release cycle.
 |------------------------------------------------------|--------------|-----------------------------|----------|
-| Start of Release Cycle                               | Release Lead | Monday 4th December 2023    | week 1   |
-| Schedule finalized                                   | Release Lead | Friday 8th December 2023    | week 1   |
-| Team finalized                                       | Release Lead | Friday 8th December 2023    | week 1   |
-| *v1.5.x & v1.6.x released*                           | Release Lead | Tuesday 9th January 2024    | week 6   |
-| *v1.5.x & v1.6.x released*                           | Release Lead | Tuesday 6th February 2024   | week 10  |
-| v1.7.0-beta.0 released                               | Release Lead | Tuesday 5th March 2024      | week 14  |
-| Communicate beta to providers                        | Comms Manager| Tuesday 5th March 2024      | week 14  |
-| *v1.5.x & v1.6.x released*                           | Release Lead | Tuesday 5th March 2024      | week 14  |
-| v1.7.0-beta.x released                               | Release Lead | Tuesday 12th March 2024     | week 15  |
-| KubeCon idle week                                    |      -       | 19th - 22nd March 2024      | week 16  |
-| release-1.7 branch created (**Begin [Code Freeze]**) | Release Lead | Tuesday 26th March 2024     | week 17  |
-| v1.7.0-rc.0 released                                 | Release Lead | Tuesday 26th March 2024     | week 17  |
-| release-1.7 jobs created                             | CI Manager   | Tuesday 26th March 2024     | week 17  |
-| v1.7.0-rc.x released                                 | Release Lead | Tuesday 2nd April 2024      | week 18  |
-| **v1.7.0 released**                                  | Release Lead | Tuesday 9th April 2024      | week 19  |
-| *v1.5.x & v1.6.x released*                           | Release Lead | Tuesday 9th April 2024      | week 19  |
+| Start of Release Cycle                               | Release Lead | Monday 11th December 2023   | week 1   |
+| Schedule finalized                                   | Release Lead | Friday 15th December 2023   | week 1   |
+| Team finalized                                       | Release Lead | Friday 15th December 2023   | week 1   |
+| *v1.5.x & v1.6.x released*                           | Release Lead | Tuesday 16th January 2024   | week 6   |
+| *v1.5.x & v1.6.x released*                           | Release Lead | Tuesday 13th February 2024  | week 10  |
+| v1.7.0-beta.0 released                               | Release Lead | Tuesday 12th March 2024     | week 14  |
+| Communicate beta to providers                        | Comms Manager| Tuesday 12th March 2024     | week 14  |
+| *v1.5.x & v1.6.x released*                           | Release Lead | Tuesday 12th March 2024     | week 14  |
+| KubeCon idle week                                    |      -       | 19th - 22nd March 2024      | week 15  |
```

Comment:
Please note: I had to flip old 1.7.0-beta.1 cutting date with Kubecon since after the week move, beta release date falls to kubecon dates